ml/Neural Network/Requirements.txt

```diff
@@ -0,0 +1,9 @@
+NOTE : I am not sure about lower version but i have this version in my pc 
+
+1 Keras - 2.3.1
+2 Scikit Learn - 0.22.1
```

Comment:
same goes with this. We will write it as :
scikit-learn==0.22.1.
Please edit this and all the libraries likewise. Amazing work :)